.github/workflows/wsl-helper-build.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
         cache: yarn
     - uses: actions/setup-go@v5
       with:
-        go-version-file: src/go/wsl-helper/go.mod
+        go-version-file: go.work
```

Comment:
Same here — this is only to build `wsl-helper`, so it makes sense to use its `go.mod`.